hack/update-generated-swagger-spec.sh

```diff
@@ -55,4 +55,18 @@ do
         os::util::sed 's|https://127.0.0.1:38443|https://127.0.0.1:8443|g' "${SWAGGER_SPEC_OUT_DIR}/${type}-${endpoint}.json"
     done
 done
+
+# Swagger 2.0 / OpenAPI docs
+curl -w "\n" "${MASTER_ADDR}/swagger.json" > "${SWAGGER_SPEC_OUT_DIR}/openshift-openapi-spec.json"
+os::util::sed 's|https://127.0.0.1:38443|https://127.0.0.1:8443|g' "${SWAGGER_SPEC_OUT_DIR}/openshift-openapi-spec.json"
```

Comment:
Everyone's got python -- can you use jsontool to do this? `sed` on JSON always sucks